src/DeviceInterface/InterfacePrototype.js

```diff
@@ -421,6 +461,8 @@ class InterfacePrototype {
         dataPromise.then(response => {
             if (response.success) {
                 return this.selectedDetail(response.success, config.type)
+            } else if (response) {
+                return this.selectedDetail(response, config.type)
```

Comment:
Is there a deviation in the usual pattern of having responses with a `success` object? What's the reasoning behind it?

Reply:
this is a carve-out because existing messages are not uniform. Once every message uses a schema-driven approach then we'd be able to remove this